src/Microsoft.Framework.PackageManager/List/DependencyListOptions.cs

```diff
@@ -0,0 +1,81 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using System.Runtime.Versioning;
+using Microsoft.Framework.Runtime.Common.CommandLine;
+using NuGet;
+
+namespace Microsoft.Framework.PackageManager.List
+{
+    internal class DependencyListOptions
+    {
+        public DependencyListOptions(Reports reports, CommandArgument path, CommandOption detailLevel, CommandOption framework)
+        {
+            // reports
+            Reports = reports;
+
+            // project
+            string projectPath = path.Value ?? Directory.GetCurrentDirectory();
+
+            Runtime.Project proj;
+
+            bool isInputValid = true;
+
+            isInputValid &= Runtime.Project.TryGetProject(projectPath, out proj);
+            Path = projectPath;
+            Project = proj;
+
+            // tree mode detail level 
+            if (detailLevel.HasValue())
+            {
+                int dl;
```

Comment:
Try not to abbreviate. This should probably be `detailLevel` whereas the parameter to this method should be `detailLevelOption` (and the other params to this method should probably also by `XyzOption` because they are not the actual values, they are the *option* that may or may not contain a value).